mammoth/model_builder.py

```diff
@@ -248,14 +247,12 @@ def create_bilingual_model(
         decoder=decoder,
         attention_bridge=attention_bridge
     )
-
     if uses_adapters(model_opts):
         logger.info('Creating adapters...')
         create_bilingual_adapters(nmt_model, model_opts, task)
     else:
         logger.info('Does not use adapters...')
     print('built model:')
-    print(nmt_model)
```

Comment:
I think you disabled this for debugging purposes, and I agree that perhaps our log-files could have different levels of verbosity (starting form printing the model in a cleaner way). For the sake of this PR, just double check if deleting this is intended :)

Reply:
I shall re-add this line